README.md

```diff
@@ -6,24 +6,24 @@ Installs AWS CloudWatch Log Agent
 Requirements
 ------------
 
-Requires ec2_facts. 
+Requires ec2_facts.
 
 Role Variables
 --------------
 
 List of logs with the following keys
 
-| Name       | Description            | Required
-|------------|------------------------|---------
-| file       | Full path to log file  | Yes
-| format     | Datetime format        | No
-| group_name | CloudWatch Log Group   | Yes
-
+| Name        | Description            | Required | Default
+|-------------|------------------------|----------|---------
+| file        | Full path to log file  | Yes      |
+| format      | Datetime format        | No       | None
+| group_name  | CloudWatch Log Group   | Yes      |
+| stream_name | CloudTrail Stream Name | No       | The instance id
```

Comment:
Looks like a small typo here, I think `CloudWatch` was intended instead of `CloudTrail`. I'd also expand it to something like `CloudWatch Log Stream Name` to match the AWS console.